### ISSUE TITLE
Cannot find 'blob' directory in your 'transformers' repository

### ISSUE THREAD

**Issue by parkhaemi** [huggingface/transformers#14495]
We are reading the paper 'EVALUATION OF NEURAL ARCHITECTURES TRAINED
WITH SQUARE LOSS VS CROSS-ENTROPY IN CLASSIFICATION TASKS' which is published as a conference paper at ICLR 2021.
In 13page, we found that 'https://github.com/huggingface/transformers/blob/master/examples/text-classification/run_glue.py'. is no longer accessed.

**Comment by seanbenhur** [comment]
It is here, https://github.com/huggingface/transformers/blob/master/examples/pytorch/text-classification/run_glue.py @parkhaemi

**Comment by parkhaemi** [comment]
Thank you so much!!
Have a nice day :)
